xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -80,7 +80,7 @@ To be able to format the output with indentation and white space OR to skip vali
       <Docs>
         <param name="utf8Json">The destination for writing JSON text.</param>
         <param name="options">Defines the customized behavior of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> By default, it writes minimized JSON (with no extra whitespace) and validates that the JSON being written is structurally valid according to the JSON RFC.</param>
-        <summary>Constructs a new <see cref="T:System.Text.Json.Utf8JsonWriter" /> instance with a specified <paramref name="utf8Json" />.</summary>
+        <summary>Constructs a new <see cref="T:System.Text.Json.Utf8JsonWriter" /> instance with a specified <see cref="T:System.IO.Stream" /> to write the output to.</summary>
```

Comment:
We normally mention streams with plain language:

```suggestion
        <summary>Constructs a new <see cref="T:System.Text.Json.Utf8JsonWriter" /> instance with a specified stream to write the output to.</summary>
```

Reply:
I'd add to Carlos suggestion and use the text from our wiki - https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Constructor

```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> class using the specified stream to write the output to and customization options.</summary>
```